clients/mobile/src/authentication/register/hooks/useRegister.ts

```diff
@@ -28,7 +29,10 @@ const validationRules: Record<CredentialKeys, ValidationRule[]> = {
 
 export const useRegister = (): any => {
   const fetchClient = useFetchClient()
+  const { userProfile } = useUserProfile()
   const { setIsAuthenticated } = useAuth()
+  const [googleLoading, setGoogleLoading] = useState(false)
+  const [facebookLoading, setFacebookLoading] = useState(false)
```

Comment:
Can't we handle loading with one variable?

Reply:
updated vai 8132c379e82ccbea66f8659a848f68ec037ace46